### PR TITLE
Responsive2

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

File: frontend/src/components/TextPhilosophy/style.jsx [hunk 1]
```diff
@@ -2,38 +2,60 @@ import styled from "styled-components";
...
     display: flex;
   }
-  .left,
-  .right {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    text-align: justify;
+
+  /*
```
Comment (LuigiMenez) [inline frontend/src/components/TextPhilosophy/style.jsx:42]: Dead code

File: frontend/src/components/TextPhilosophy/style.jsx [hunk 2]
```diff
@@ -42,16 +64,40 @@ export default styled.section`
...
     }
+
+    .descriptionText {
+      width: 100%;
+      p {
+        padding: 0.3em;
+        text-align: start;
+      }
+    }
+    /*
```
Comment (LuigiMenez) [inline frontend/src/components/TextPhilosophy/style.jsx:100]: Same

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Dead code

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**